SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorLayoutDetails.cpp

```diff
@@ -124,65 +126,20 @@ void FSpatialGDKEditorLayoutDetails::CustomizeDetails(IDetailLayoutBuilder& Deta
 
 FReply FSpatialGDKEditorLayoutDetails::GenerateDevAuthToken()
 {
-	FString Arguments = TEXT("project auth dev-auth-token create --description=\"Unreal GDK Token\" --json_output");
-	if (GetDefault<USpatialGDKSettings>()->IsRunningInChina())
-	{
-		Arguments += TEXT(" --environment cn-production");
-	}
-
-	FString CreateDevAuthTokenResult;
-	int32 ExitCode;
-	FSpatialGDKServicesModule::ExecuteAndReadOutput(SpatialGDKServicesConstants::SpatialExe, Arguments, SpatialGDKServicesConstants::SpatialOSDirectory, CreateDevAuthTokenResult, ExitCode);
-
-	if (ExitCode != 0)
+	FString DevAuthToken;
+	FString ErrorMessage;
+	if (!SpatialCommandUtils::GenerateDevAuthToken(GetDefault<USpatialGDKSettings>()->IsRunningInChina(), DevAuthToken, ErrorMessage))
```

Comment:
Just interested why we don't use FSpatialGDKDevAuthTokenGenerator here.

Reply:
This is a button that is there specifically for generating the dev auth token so it's more straightforward to just generate it synchronously in this case.